src/sim/moonbot_isaac/environments/observer_camera.py

```diff
@@ -0,0 +1,121 @@
+import omni.graph.core as og
```

Comment:
@azazdeaz I think `omni` is being deprecated for `isaacsim` if i'm not mistaken. 
Not this PR, but I guess at somepoint we should rename everything

Reply:
Good point, thanks! But this API (AFAIK) will stay because it's not part of Isaac Sim.